test/sql/cloud/hierarchical_namespace.test

```diff
@@ -102,17 +102,16 @@ statement ok
 SET azure_http_stats = true;
 
 query II
-EXPLAIN ANALYZE SELECT count(*) FROM 'abfss://${DUCKDB_AZ_DATA_DIR}/partitioned/l_receipmonth=*7/l_shipmode=TRUCK/*.csv';
+EXPLAIN ANALYZE SELECT count(*) FROM 'abfss://${ABFSS_DATA_DIR}/partitioned/l_receipmonth=*7/l_shipmode=TRUCK/*.csv';
 ----
 analyzed_plan	<REGEX>:.*HTTP Stats.*in\: [\d.]* KiB.*\#HEAD\: 1.*GET\: 4.*PUT\: 0.*\#POST\: 0.*
 
 query II
-EXPLAIN ANALYZE SELECT count(*) FROM 'abfs://${DUCKDB_AZ_DATA_DIR}/partitioned/l_receipmonth=*7/l_shipmode=TRUCK/*.csv';
+EXPLAIN ANALYZE SELECT count(*) FROM 'abfs://${ABFSS_DATA_DIR}/partitioned/l_receipmonth=*7/l_shipmode=TRUCK/*.csv';
 ----
 analyzed_plan	<REGEX>:.*HTTP Stats.*in\: [\d.]* KiB.*\#HEAD\: 1.*GET\: 4.*PUT\: 0.*\#POST\: 0.*
 
-
-query II
-EXPLAIN ANALYZE SELECT count(*) FROM 'azure://${DUCKDB_AZ_DATA_DIR}/partitioned/l_receipmonth=*7/l_shipmode=TRUCK/*.csv';
-----
-analyzed_plan	<REGEX>:.*HTTP Stats.*in\: [\d.]* KiB.*\#HEAD\: 1.*GET\: 2.*PUT\: 0.*\#POST\: 0.*
+#query II
+#EXPLAIN ANALYZE SELECT count(*) FROM 'azure://${AZ_DATA_DIR}/partitioned/l_receipmonth=*7/l_shipmode=TRUCK/*.csv';
+#----
+#analyzed_plan	<REGEX>:.*HTTP Stats.*in\: [\d.]* KiB.*\#HEAD\: 1.*GET\: 2.*PUT\: 0.*\#POST\: 0.*
```

Comment:
What's the reason to comment this out? I think we either need to fix this, remove this, or explain why it's commented out